### PR TITLE
The necropolis tendril is anchored again.

### LINKED ISSUE #29441: You can hand-tele Necropolis Tendrils on board the station
Meteorshot was used to knock it through.

### LINKED ISSUE #40015: You can move the necropolis one tile by throwing and hitting it with a body.
Issue reported from  Round ID: 93208 (/tg/Station Bagil [ENGLISH] [US-EAST] [100% FREE LAG])

Reporting client version: 512

### PR DESCRIPTION
**Pull request by harmonyn**
:cl: bandit
fix: The necropolis tendril is anchored again.
/:cl:

Fixes half of https://github.com/tgstation/tgstation/issues/40520.
Fixes #40015
Fixes #29441 

it turns out the proc for being hit by a thrown thing doesn't give a fuck about move_resist, this seems like the more reasonable solution though than rewriting throwing again

@kevinz000

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/necropolis_tendril.dm [hunk 1]
```diff
@@ -22,6 +22,7 @@
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
 	maxbodytemp = INFINITY
+	anchored = 1
```
Comment (Barhandar) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/necropolis_tendril.dm]: `TRUE`

**Comment by Jordie0608** [comment]
This is causing master to fail the anchored mobs unit test.

**Comment by Cyberboss** [comment]
>Not respecting the plumber

**Comment by ninjanomnom** [comment]
lol I just now looked at this pr. Mobs need to not be using anchored because it simplifies other code elsewhere. Respect the plumber.

**Pull request merged**